search-provider/eks-discovery-feed-provider.c

```diff
@@ -283,6 +313,19 @@ add_key_value_pair_from_model_to_variant (EkncContentObjectModel *model,
   add_key_value_pair_to_variant (builder, underscore_key, value);
 }
 
+static void
+add_key_int_value_pair_from_model_to_variant (EkncContentObjectModel *model,
```

Comment:
That's true, it is a little odd. I guess the intention here was to "read an int from the model and serialise it to a string". Having to do the int-reading outside the function would make it a lot less convenient ... that said, this can be solved by using another function :)